rascal-textmate-core/src/main/rascal/lang/rascal/grammar/Util.rsc

```diff
@@ -31,15 +31,6 @@ bool tryParse(Grammar g, Symbol s, str input, bool allowAmbiguity = false) {
     return false;
 }
 
-@synopsis{
-    Gets the terminals that occur in production `p`, possibly recursively
-    (default: `true`)
-}
-
-set[Symbol] getTerminals(Grammar g, Production p, bool recur = true)
```

Comment:
This function was used before this PR to compute individual symbols to gobble up between `begin`/`end`, but it's not longer needed